utils/arguments.py

```diff
@@ -0,0 +1,8 @@
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--support-brownie",
+    help="Support recursive retrieving for contracts based on brownie framework.",
```

Comment:
Please provide more context in the help description regarding the use case. Could take some info from https://github.com/lidofinance/diffyscan/pull/25#issuecomment-1777658907